packages/components/src/components/List/UnorderedList.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from "@xstyled/styled-components";
+import React, { ReactElement } from "react";
+import { Box } from "../../primitives/Box";
+import { ListItemProps } from "./ListItem";
+
+export type ListFontColors = "colorText" | "colorTextLink";
+export interface UnorderedListProps
+  extends Omit<React.HTMLAttributes<HTMLUListElement>, "color"> {
```

Comment:
Also, we don't need to do this anymore.

Reply:
We need it because the `Box.ul` has color and also the `HTMLUListElement` and they are incompatible.